kubehub/migrations/0003_auto_20200305_0817.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.3 on 2020-03-05 08:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kubehub', '0002_auto_20200226_1524'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='KubesprayDeploy',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
not good, where is vm_group id we are deploying to?
where is cluster id we are deploying
don't store path to file, just use vm_group id and kubespray deploy id in a file name.

Reply:
fixed in 77a44cc
- used vm_group id and kubespray deploy id in a file name instead of time zone